cpu/esp_common/esp-riscv/exceptions.c

```diff
@@ -70,6 +106,14 @@ void panic_arch(void)
                    _frame->mepc, exceptions[_frame->mcause]);
     }
 #if defined(DEVELHELP)
+    /* print heap statistics */
     heap_stats();
+    /* break in debugger if attached or busy wait until WDT resets the MCU */
+    if (cpu_ll_is_debugger_attached()) {
+        cpu_ll_break();
+    }
+    else {
+        while (1) { }
```

Comment:
maybe 
```suggestion
        while (IS_USED(DEVELHELP)) { }
        pm_reboot();
```

Reply:
It isn't necessary to use it here because this part of the code is only compiled if `DEVELHELP` is defined because of the conditional in line #109.